src/components/PageTitle/PageTitle.test.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import PageTitle from "./PageTitle";
+import { render } from "@testing-library/react";
+
+describe("<PageTitle />", function () {
+  it("should match the snapshot", () => {
+    /**
+     * Write a snapsho◘t test for PageTitle component
+     * Don't forget to add snapshot file to the git
+     */
+
+    const labelText = "label text";
+    const { container, getByText } = render(<PageTitle label={labelText} />);
+
+    expect(getByText(labelText)).toBeInTheDocument();
```

Comment:
You have already checked that label was rendered via snapshot test, so it is not necessary to check the labelText - it's just a double work, or you do a snapshot or you check by queries

Reply:
got it. Tx